gnomad_qc/v5/data_ingestion/field_requirements.md

```diff
@@ -54,54 +54,56 @@ All histograms annotations must use the same bin edges as defined in their respe
-| **rsid**                                                                                  | `set<str>`                                                            | dbSNP reference SNP identification (rsID) numbers.                                                                   | `{"rs123", "rs456"}`                           | not_needed      | will re-annotate                                                                                                                                                   |
-| **filters**                                                                               | `set<str>`                                                            | Variant filters; AC0: Allele count is zero after filtering out low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls), AS_VQSR: Failed allele-specific VQSR filtering thresholds of -4.0598 for SNPs and 0.1078 for indels, InbreedingCoeff: GATK InbreedingCoeff < -0.3. An empty set in this field indicates that the variant passed all variant filters.                                                                    | `{"PASS"}`                                     | optional        |AC0: The allele count is zero after filtering out low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls).                                                                                                                                                                     |
+| **rsid**                                                                                  | `set<str>`                                                            | dbSNP reference SNP identification (rsID) numbers.                                                                   | -                           | not_needed      | will re-annotate                                                                                                                                                   |
+| **filters**                                                                               | `set<str>`                                                            | Variant filters; 'AC0': Allele count is zero after filtering out low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls), 'AS_VQSR': Failed allele-specific VQSR filtering thresholds, 'InbreedingCoeff': GATK InbreedingCoeff < -0.3. An empty set in this field indicates that the variant passed all variant filters.                                                                    | `{"PASS"}`                                     | optional        |AC0: The allele count is zero after filtering out low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls).                                                                                                                                                                     |
```

Comment:
I know you said the examples aren't accurate yet, but just wanted to flag that this says an empty set indicates a variant passed all filters but the example says `{PASS}`